_includes/use-cases/use-case-consumption-accept-incoming-request.md

```diff
@@ -0,0 +1,23 @@
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use-case attempts to accept an incoming [Request](/integrate/data-model-overview#request).
```

Comment:
We render links to other pages in this documentation using jekyll's liquid: `{% link _docs_integrate/data-model-overview.md %}`.
So please check all occurrences of `](/` (all introduced by you) and replace them with the generator.

In this case it would be:
```suggestion
This use-case attempts to accept an incoming [Request]({% link _docs_integrate/data-model-overview.md %}#request).
```

Reply:
And before this knowledge gets lost again, please add this to the README.